CMakeLists.txt

```diff
@@ -705,9 +705,14 @@ add_definitions(-DQT_DISABLE_DEPRECATED_BEFORE=0x050800)
 # For fast string concatenation
 add_definitions(-DQT_USE_QSTRINGBUILDER)
 
-if (WITH_GEOREFERENCER OR WITH_ANALYSIS)
+set(WITH_GSL TRUE CACHE BOOL "Determines whether GSL library should be used")
+
+if (WITH_ANALYSIS AND WITH_GSL)
```

Comment:
```suggestion
if ((WITH_ANALYSIS OR WITH_GEOREFERENCER) AND WITH_GSL)
```

Reply:
@m-kuhn , I think that's wrong, because if we don't build with analysis, with can't build with georeferencer. So it's not an analysis _or_ georeferencer scenario here.

Reply:
I only find WITH_GEOREF in app

Reply:
That's correct. The presence (or absence) of the georeferencer itself is in app, via HAVE_GEOREFENCER flag. That flag is set when WITH_GEOREFERENCER as well as WITH_ANALYSIS & WITH_GSL are all true.

Reply:
Shall we get rid of WITH_GEOREFERENCER in this case? It seems mostly redundant now.

Reply:
@m-kuhn , done.